src/js/view/controls/rightclick.js

```diff
@@ -119,13 +124,10 @@ export default class RightClick {
         this.addOffListener(document);
 
         // Track if the mouse is above the menu or not
-        this.elementUI = new UI(this.el, { useHover: true })
-            .on('over', function() {
-                this.mouseOverContext = true;
-            }, this)
-            .on('out', function() {
-                this.mouseOverContext = false;
-            }, this);
+        this.onMouseOverHandler = this.onMouseOver.bind(this);
+        this.onMouseOutHandler = this.onMouseOut.bind(this);
+        this.el.addEventListener('mouseover', this.onMouseOverHandler);
+        this.el.addEventListener('mouseout', this.onMouseOutHandler);
```

Comment:
Since these are only called on this element to toggle an instance property, and this element is never reused, I would suggest we just add arrow listeners and let GC take care of the cleanup:

```js
this.el.addEventListener('mouseover', () => {
  this.mouseOverContext = true
});
this.el.addEventListener('mouseout', () => {
  this.mouseOverContext = false
});
```